stable/artifactory-ha/values.yaml

```diff
@@ -114,6 +114,15 @@ artifactory:
     # version:
     pullPolicy: IfNotPresent
 
+  loggers: []  # add any of the loggers to a sidecar if you want to be able to see them with kubectl logs or a log collector in your k8s cluster
+  # - request
+  # - event
+  # - binarystore
+  # - request_trace
+  # - access
+  # - artifactory
+  # - build_info_migration
+
```

Comment:
Can you also add the tomcat logs? They are under the `logs/catalina` directory, and this feature will be very useful for collecting them too.
For this you will also need to update the `{{ . | replace "_" "-" }}` to care for `/`.

Reply:
The problem with this is there does not appear to be a fixed file name where the most recent logs are. When I ls in the container I see:

```
/data/logs/catalina # ls
catalina.2019-02-07.log      catalina.2019-02-09.log      host-manager.2019-02-08.log  localhost.2019-02-07.log     localhost.2019-02-09.log     manager.2019-02-08.log
catalina.2019-02-08.log      host-manager.2019-02-07.log  host-manager.2019-02-09.log  localhost.2019-02-08.log     manager.2019-02-07.log       manager.2019-02-09.log
```

Reply:
We could do tail -f catalina.*.log, but that will make the line-level output messy when the file changes, also unsure whether it will pick up new files.

Reply:
You are right. It will not pick up new files. Only existing. We'll try to improve it on our side...